pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -435,6 +466,20 @@ class BinaryContent:
     kind: Literal['binary'] = 'binary'
     """Type identifier, this is available on all parts as a discriminator."""
 
+    def __init__(
+        self,
+        data: bytes,
+        media_type: AudioMediaType | ImageMediaType | DocumentMediaType | str,
```

Comment:
Please add a `*` ahead of `media_type` so all arguments other than `url` need to be keywords arguments